docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -264,30 +356,36 @@ spec:
         spec:
           tls:
             - hosts:
-                - ing.sslacceptor.send-receive-0.send-receive-project.demo.artemiscloud.io
+                - ing.sslacceptor.send-receive-0.send-receive-project.${CLUSTER_IP}.nip.io
               secretName: send-receive-sslacceptor-0-svc-ing-ptls
   brokerProperties:
     - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
     - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
     - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
 EOF
```

Comment:
Defining the variable INGRESS_HOST you don't need to split the writing of the file deploy.yml
```
INGRESS_HOST='ing.$(ITEM_NAME).$(CR_NAME)-$(BROKER_ORDINAL).$(CR_NAMESPACE).$(INGRESS_DOMAIN)'
kubectl apply -f - << EOF
apiVersion: broker.amq.io/v1beta1
kind: ActiveMQArtemis
metadata:
  name: send-receive
  namespace: send-receive-project
spec:
  ingressDomain: demo.artemiscloud.io
  ingressDomain: ${CLUSTER_IP}.nip.io
  acceptors:
    - name: sslacceptor
      port: 62626
      sslEnabled: true
      expose: true
      exposeMode: ingress
      sslSecret: send-receive-sslacceptor-0-svc-ing-ptls
      ingressHost: ${INGRESS_HOST}
  resourceTemplates:
    - selector:
        kind: Ingress
@@ -264,30 +356,36 @@ spec:
        spec:
          tls:
            - hosts:
                - ing.sslacceptor.send-receive-0.send-receive-project.demo.artemiscloud.io
                - ing.sslacceptor.send-receive-0.send-receive-project.${CLUSTER_IP}.nip.io
              secretName: send-receive-sslacceptor-0-svc-ing-ptls
  brokerProperties:
    - addressConfigurations."APP.JOBS".routingTypes=ANYCAST
    - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
    - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
EOF
```